test/DocumentFormat.OpenXml.Packaging.Tests/OpenXmlPackageTests.cs

```diff
@@ -261,5 +261,24 @@ public void TestDataReferenceRelationshipsAreClonedCorrectly()
             MediaReferenceRelationship mrr = slidePart.AddMediaReferenceRelationship(mediaDataPart, "rId4");
             Assert.Equal("rId4", mrr.Id);
         }
+
+        // This is a regression test for issue #1069
+        // Opening a PowerPoint presentation with a 3d model graphic will fail if the Model3dReferenceRelationshipPart doesn't
+        //     accept both model/gltf-binary *and* model/gltf.binary MIME types. PowerPoint writes the latter.
+        [Fact]
+        public void TestOpenModel3DWrittenByPowerPoint()
+        {
+            using (var testFile = GetStream(TestFiles._3DTestDash, false))
+            using (var presDoc = PresentationDocument.Open(testFile, false))
+            {
+                Assert.NotNull(presDoc);
+            }
+
+            using (var testFile = GetStream(TestFiles._3DTestDot, false))
```

Comment:
Can these be separate tests? Also, it would nice to assert on the content type of the package

Reply:
@twsouthwick not sure what you mean by "content type of the package". Do you mean assert on the model/gltf-binary content type (the part) or do you mean something else?  The test is pretty much about validating the model/gltf(-|.)binary content type so I don't know why it should explicitly assert this. Maybe you mean assert that the glb part is not something else?

Reply:
Because then when you look at the test, you know **that** is what you're testing here. In a year, or 5 years, you may not remember exactly what the test is supposed to be doing

Reply:
what i meant was, there is no content type of the package. do you mean assert on the content type of the .glb part? If so, I can probably add code to find the part and check that it's either model/gltf-binary or model/gltf.binary.  But we say that in the comments and the code will assert that naturally.

Reply:
But I can see what you mean. It's not clear what the test is actually looking for other than a clean open with a non-null package object. I'll add some code to look for the part and see if it's one of the two.

Reply:
Actually, it will have to be model/gltf-binary. The part class (Model3DReferenceRelationshipPart) will always show as model/gltf-binary internally and will write this. But will just look the other way when loading model/glft.binary. I added the assert for it.